apps/mobile/src/components/amount-field/amount-field.tsx

```diff
@@ -1,45 +1,53 @@
 import { AmountSendMaxButton } from '@/components/amount-field/amount-send-max-button';
 
+import { CryptoCurrency, FiatCurrency, MarketData } from '@leather.io/models';
 import { Box, Theme } from '@leather.io/ui/native';
 
 import { AmountFieldPrimaryValue } from './amount-field-primary-value';
 import { AmountFieldSecondaryValue } from './amount-field-secondary-value';
 
-type CurrencyMode = 'crypto' | 'fiat'; // TODO: Should be moved into containing form types
+type CurrencyMode = 'crypto' | 'fiat';
```

Comment:
This type is duplicated in your temporary types. 

I think that this PR is difficult to review as you still have a lot of files in a new folder - `send-refactor` and you're not yet deleting the things that they are replacing. 

This makes it difficult to tell if it's really improving things or if there are some things that are missing

Reply:
This is intentional, I'd rather not couple the amount field to Send form types, as it will likely be used in at least the Swap form too. I might end up replacing this with a boolean `isFiatEntry` when bringing back the fiat entry.

Reply:
In hindsight, made this a global type. Details [here](https://github.com/leather-io/mono/pull/984/files/fd83e6cc86c5f4e8f396de1bd34cca46fe528665#r2009694420)